integration-tests/tests/register-user.js

```diff
@@ -23,7 +27,15 @@ const testRegisterUserFn = withUser(async ({ page, testUrl, token }) => {
   // We wait here for the cookie message to disappear, it has a fixed duration fading animation of 300ms
   await delay(350);
   await verifyAccessibility(page);
-  await click(page, clickable({ textContains: 'View Workspaces' }));
+
+  // This is the hamburger menu
+  await click(page, '/html/body/div[1]/div[2]/div/div[1]/div[1]/div[1]/div/div[1]');
```

Comment:
There has to be a better way!! I would recommend either (in order of preference):

1) Adding an aria-label to the button and using that to select it.
2) Adding a unique ID to the button and using that.
3) Using the button's aria-role to get it (this one could be hard because there are likely multiple such items on the page).